index.js

```diff
@@ -17,11 +19,31 @@ function bookSearch () {
 
       books = JSON.parse(JSON.stringify(data.items));
 
-      console.log(data);
+      console.log(data.items[0]);
     },
     type: 'GET'
   })
 }
+function bookMore () {
+  const search = document.getElementById('search').value;
+
+  start += step;
+
+  $.ajax({
```

Comment:
Move general part of url to the constant variable (url). It will look more pretty ;-)